c2rust-transpile/src/cfg/inc_cleanup.rs

```diff
@@ -136,32 +130,28 @@ impl IncCleanup {
 /// Remove empty else clauses from if expressions that can arise from
 /// removing idempotent statements.
 fn cleanup_if(stmt: Stmt) -> Stmt {
-    if let Stmt::Expr(ref expr) = &stmt {
-        if let Expr::If(ExprIf {
-            ref cond,
-            then_branch: ref body,
-            else_branch: ref els,
-            ..
-        }) = *expr
-        {
-            if let Some((_token, ref els)) = *els {
-                if let Expr::Block(ExprBlock {
-                    block: ref blk,
-                    label: None,
-                    ..
-                }) = **els
-                {
-                    if blk.stmts.is_empty() {
-                        return Stmt::Expr(Expr::If(ExprIf {
-                            attrs: vec![],
-                            if_token: Default::default(),
-                            cond: cond.clone(),
-                            then_branch: body.clone(),
-                            else_branch: None,
-                        }));
-                    }
-                }
+    if let Stmt::Expr(Expr::If(ExprIf {
+        cond,
+        then_branch: body,
+        else_branch: Some((_token, block)),
+        ..
+    })) = &stmt
+    {
+        match &**block {
+            Expr::Block(ExprBlock {
+                block: blk,
```

Comment:
No need to rename `block`.

Reply:
Fixed by https://github.com/immunant/c2rust/pull/460/commits/4b4257d8b69048d9012e44db027d3104ab2ca2c3.